vital/types/descriptor_set.cxx

```diff
@@ -0,0 +1,155 @@
+/*ckwg +29
+ * Copyright 2013-2018 by Kitware, Inc.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *  * Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ *  * Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ *  * Neither name of Kitware, Inc. nor the names of any contributors may be used
+ *    to endorse or promote products derived from this software without specific
+ *    prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS ``AS IS''
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHORS OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * \file
+ * \brief core descriptor_set interface implementation
+ */
+
+#include "descriptor_set.h"
+
+#include <sstream>
+
+namespace kwiver {
+namespace vital {
+
+  // ----------------------------------------------------------------------------
+descriptor_set
+::descriptor_set()
+  : m_logger( kwiver::vital::get_logger( "vital.descriptor_set" ) )
+{
+}
+
+descriptor_set
+::~descriptor_set()
+{ }
+
+// ----------------------------------------------------------------------------
+kwiver::vital::logger_handle_t
+descriptor_set::
+logger()
+{
+  return m_logger;
+}
+
+// ============================================================================
+// Constructor from a vector of descriptors
+simple_descriptor_set
```

Comment:
Curious. There is a new image_container_set_simple, why don't we have the same for the simple_descriptor_set? Seems cleaner than jamming that code into the descriptor_set.

Reply:
Looks like @linus-sherrill collapsed the separated ``simple_image_container_set`` into the ``vital/types/image_container_set.h/cxx" files.